core-bundle/src/Security/User/ContaoUserProvider.php

```diff
@@ -115,8 +115,7 @@ private function validateSessionLifetime(User $user): void
             return;
         }
 
-        $config = $this->framework->getAdapter(Config::class);
-        $timeout = (int) $config->get('sessionTimeout');
+        $timeout = (int) ($this->sessionStorageOptions['gc_maxlifetime'] ?? \ini_get('session.gc_maxlifetime'));
```

Comment:
Again, I do not think we need that at all. Since we are not using `localconfig.sessionTimeout` anymore we can remove this method altogether.

- If you define `framework.session.gc_maxlifetime`, Symfony will execute `ini_set('session.gc_maxlifetime`).
- If you define `framework.session.cookie_lifetime`, Symfony will execute `ini_set('session.cookie_lifetime`).

Either way, the session will time out without our `ContaoUserProvider` checking this setting.

And if you use the `RedisSessionHandler` for example, the session timeout will be defined by the `ttl` option in the service arguments.

Reply:
I agree that this does not make sense. What would make sense is to have a custom session lifetime setting for BE and FE users (imho that's a feature that's been requested before). This one could reduce the lifetime of the server side setting. It cannot increase it for obvious reasons, though.

Reply:
I‘d vote for removing it, but I‘m fine either way.

Reply:
Maybe the Symfony security experts should decide then? @bytehead @aschempp ?

Reply:
I think the point is as @Toflar said to have a custom session timeout. It's a feature that standard Symfony does not seem to have, but we added because Contao 3 had it. Not sure if it is really necessary, but I know there have been reports about it not working (= people at least try to use it). I'm fine either way, I think it can still be added, but I would not go for `gc_maxlifetime`.

Reply:
Removed in 183c92265bc0dbc0f403e7664dbc28a8782a666f